install/kubernetes/helm/istio/README.md

```diff
@@ -58,12 +58,9 @@ $ helm init --service-account tiller
 
 4. If you are enabling `kiali`, you need to create the secret that contains the username and passphrase for `kiali` dashboard:
    ```
-   $ echo -n 'admin' | base64
-   YWRtaW4=
-   $ echo -n '1f2d1e2e67df' | base64
-   MWYyZDFlMmU2N2Rm
-   $ NAMESPACE=istio-system
-   $ cat <<EOF | kubectl apply -f -
+   NAMESPACE=istio-system
```

Comment:
You may want to rebase, as the namespace creation was moved to step 3 at https://github.com/istio/istio/commit/536df4b8cb473b297adfdb3855e427c0ec80a63f

Reply:
conflict resolved